packages/wallet/test/services/DelegationTracker/transactionCertificates.test.ts

```diff
@@ -0,0 +1,94 @@
+import { Cardano } from '@cardano-sdk/core';
+import {
+  TransactionsTracker,
+  isLastStakeKeyCertOfType,
+  outgoingTransactionsWithCertificates,
+  transactionHasAnyCertificate,
+  transactionStakeKeyCertficates
+} from '../../../src';
+import { createTestScheduler } from '../../testScheduler';
+
+describe('transactionCertificates', () => {
+  test('transactionStakeKeyCertficates', () => {
+    const certificates = transactionStakeKeyCertficates({
+      certificates: [
+        { __typename: Cardano.CertificateType.StakeDelegation } as Cardano.Certificate,
```

Comment:
I've just picked this up when reviewing the tests. `CertificateType.StakeRegistration` should be `CertificateType.StakeKeyRegistration` for clarity, ditto for deregistration. Without it we're indicating the stake is being registered, which could be confused with the concept of delegation.